examples/rust/src/main.rs

```diff
@@ -164,7 +164,7 @@ async fn main() -> Result<()> {
     let team_id = team
         .owner
         .client
-        .create_team()
+        .create_team(TeamConfigBuilder::new().build()?)
```

Comment:
@nikki-aranya, @jdygert-spok there were some hoops we needed to jump through to get the C API builder pattern working, but we should be using an idiomatic builder pattern in Rust if possible. E.g.:
```suggestion
        .create_team(TeamConfigBuilder::new()?)
```

Reply:
if/when that actually has parameters, it'll be split out into a let block